File: 2185. Counting Words With a Given Prefix/solution.js
/**
 * @param {string[]} words
 * @param {string} pref
 * @return {number}
 */
var prefixCount = function(words, pref) {
    let sum = 0;
    for (const word of words){
        if(word.startsWith(pref)){
            ++sum;
        }
    }
    return sum;
};

File: 2185. Counting Words With a Given Prefix/solution.c
int prefixCount(char ** words, int wordsSize, char * pref){
    int sum = 0;
    int len = strlen(pref);
    for (int id = 0; id < wordsSize;++id){
        if(strncmp(words[id], pref, len) == 0){
            ++sum;
        }
    }
    return sum;
}